dockerfiles/idp_ci.Dockerfile

```diff
@@ -20,19 +20,14 @@ RUN apt-get update -qq && \
     python3-venv \
     nodejs \
     locales \
-    # google-chrome-stable \
+    google-chrome-stable \
     yarn
 
-# This is temporary so that we do not use the latest chrome due to an issue in the latest version
-ARG CHROME_VERSION="112.0.5615.165-1"
-RUN wget --no-verbose -O /tmp/chrome.deb https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_VERSION}_amd64.deb \
-  && apt install -y /tmp/chrome.deb \
-  && rm /tmp/chrome.deb
-# This is temporary so that we do not use the latest chromedriver due to an issue in the latest version
-# RUN curl -Ss "https://chromedriver.storage.googleapis.com/$(google-chrome --version | grep -Po '\d+\.\d+\.\d+' | tr -d '\n').16/chromedriver_linux64.zip" > /tmp/chromedriver.zip && \
-RUN curl -Ss "https://chromedriver.storage.googleapis.com/$(curl -Ss "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$(google-chrome --version | grep -Po '\d+\.\d+\.\d+' | tr -d '\n')")/chromedriver_linux64.zip" > /tmp/chromedriver.zip && \
+RUN google-chrome --version
+
```

Comment:
Did we want this to stay, or was it for debugging?

```suggestion
```

Reply:
Oops, debugging!